docs/docs/1.5.x/installation/docker.md

```diff
@@ -91,7 +89,7 @@ mtls:
   backends:
   - name: ca-1
     type: builtin" | docker run -i --net="host" \
-  docker.io/kumahq/kumactl: kumactl apply -f -
+  docker.io/kumahq/kumactl:<version> kumactl apply -f -
```

Comment:
I guess we can't use the template variable for the current version?

Reply:
Yep I've spent an hour trying to figure out how to do this with no success :( It's a limitation of vuepess